pkg/cmd/tracerunner.go

```diff
@@ -13,15 +13,51 @@ import (
 	"syscall"
 
 	"github.com/fntlnz/mountinfo"
+	"github.com/iovisor/kubectl-trace/pkg/tracejob"
 	"github.com/spf13/cobra"
 )
 
+const (
+	bpftrace = "bpftrace"
+	bcc      = "bcc"
+)
+
+var (
+	bpfTraceBinaryPath = "/usr/bin/bpftrace"
+)
+
 type TraceRunnerOptions struct {
-	podUID             string
-	containerName      string
-	inPod              bool
-	programPath        string
-	bpftraceBinaryPath string
+	// The tracing system to use.
+	// tracer = bpftrace | bcc | perf
+	tracer string
+
+	// Selector (label query) that identifies entity to be traced.
+	// selector = label '=' value [',' labelN '=' valueN ...]
+	// Currently supported labels:
```

Comment:
Is it possible to put these currently supported values as part of the command line parsing lib so that if someone messes up they get a nice error message?

Reply:
Will do, see my previous comment regarding breaking change.